qa/workunits/deepsea/common/deploy.sh

```diff
@@ -46,29 +46,14 @@ function _initialize_minion_array {
     if type salt-key > /dev/null 2>&1; then
         MINION_LIST=$(salt-key -L -l acc | grep -v '^Accepted Keys')
         for m in $MINION_LIST ; do
-            echo "Adding minion $m to minion array"
             MINION_ARRAY[0]=$m
             i=$((i+1))
         done
     else
         echo "Cannot find salt-key. Is Salt installed? Is this running on the Salt Master?"
         exit 1
     fi
-    echo "There are $i minions in this Salt cluster"
-}
-
-function _update_salt {
-    # make sure we are running the latest Salt before Stage 0 starts,
-    # otherwise Stage 0 will update Salt and then fail with cryptic
-    # error messages
```

Comment:
Explanation why this is being removed:

When https://github.com/SUSE/DeepSea/pull/1431 is merged, this will no longer be needed in *any* environment, and anyway the SES6 CI now guarantees the latest salt RPMs (for a given set of repos) are installed and running in the test environment.